modules/nf-core/matrixfilter/meta.yml

```diff
@@ -0,0 +1,71 @@
+name: "matrixfilter"
+description: filter a matrix based on a minimum value and numbers of samples
+  that must pass.
+keywords:
+  - matrix
+  - filter
+tools:
+  - "matrixfilter":
+      description: "filter a matrix based on a minimum value and numbers of samples"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing information on experiment, at a minimum an id.
+        e.g. [ id:'test' ]
+  - samplesheet:
+      type: file
+      description: |
+        CSV or TSV format sample sheet with sample metadata
```

Comment:
I would make this a separate optional input, I could imagine people just want to filter based on the entire dataset.

Reply:
I actually started off doing it that way, but decided it was an unnecessary complication to the logic given that something like a sample sheet will always be around. But I can re-instate it.